certbot/renewal.py

```diff
@@ -38,6 +38,9 @@ def renewal_conf_files(config):
     """Return /path/to/*.conf in the renewal conf directory"""
     return glob.glob(os.path.join(config.renewal_configs_dir, "*.conf"))
 
+def renewal_file_for_certname(config, certname):
+    """Return /path/to/certname.conf in the renewal conf directory"""
+    return os.path.join(config.renewal_configs_dir, "{0}.conf".format(certname))
```

Comment:
This should check whether the file exists and is readable, and return a sensible error if it doesn't / isn't.  Currently, if you try to renew a non-existent name, you get:

```
certbot renew --cert-name nonexistentname
Saving debug log to /var/log/letsencrypt/letsencrypt.log

-------------------------------------------------------------------------------
Processing /etc/letsencrypt/renewal/nonexistentname.conf
-------------------------------------------------------------------------------
renewal config file {} is missing a required file reference
Renewal configuration file /etc/letsencrypt/renewal/nonexistentname.conf is broken. Skipping.

No renewals were attempted.

Additionally, the following renewal configuration files were invalid: 
  /etc/letsencrypt/renewal/nonexistentname.conf (parsefail)
0 renew failure(s), 1 parse failure(s)
```

When perhaps a more succinct error would do?